apps/site/next-data/providers/blogData.ts

```diff
@@ -1,12 +1,19 @@
 import { cache } from 'react';
 
-import generateBlogData from '@/next-data/generators/blogData.mjs';
 import { BLOG_POSTS_PER_PAGE } from '@/next.constants.mjs';
+import { blogData } from '@/next.json.mjs';
 import type { BlogCategory, BlogPostsRSC } from '@/types';
 
-const { categories, posts } = await generateBlogData();
+function getPosts() {
+  return blogData.posts.map(post => ({
+    ...post,
+    date: new Date(post.date),
+  }));
+}
 
-export const provideBlogCategories = cache(() => categories);
+const posts = getPosts();
+
+export const provideBlogCategories = cache(() => blogData.categories);
```

Comment:
https://github.com/nodejs/nodejs.org/blob/862c078202461b5f76583e95251d5948272f446b/apps/site/next.dynamic.constants.mjs#L4

blogCategories are only used here, I would rather simply import blogData from the file. No need to have this whole provideBlogCategories freakery.

Reply:
agreed 👍

Reply:
updated :slightly_smiling_face: